ttnn/cpp/ttnn/operations/experimental/transformer/nlp_create_qkv_heads/nlp_create_qkv_heads_pybind.hpp

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+
+namespace ttnn::operations::experimental::transformer::detail {
+
+namespace py = pybind11;
+void bind_nlp_create_qkv_heads(py::module& module);
```

Comment:
see cll/all_gather , it gives an example of using pybind_fwd.hpp

Reply:
Done